extension/src/popup/components/signTransaction/Operations/index.tsx

```diff
@@ -85,6 +92,10 @@ interface TransactionInfoResponse {
   sponsoredId: string;
   trustor: string;
   type: keyof typeof OPERATION_TYPES;
+  value: {
+    type: string;
+    data: Buffer;
+  };
```

Comment:
here we're just adding some possible operation fields that we had missed in the past